web/thesauruses/_meta/io.json

```diff
@@ -0,0 +1,243 @@
+{
+    "meta": {
+        "language": "language_id",
+        "language_version": "version.number",
+        "language_name": "Human-Friendly Language Name"
+    },
+    "categories": {
```

Comment:
I'm kind of wondering if each section should have like a statement to include. Like C++ has `#include <iostream>`, `#include <fstream>`, etc. to import that functionality. I know Java has it, Python has imports for like networking stuff, PHP has it all built-in though. So this might be a thing we should include.

Reply:
Added to everything but console as that is normally included in the base library for many languages.